pkg/controller/handlers/poweruserworkspace/poweruserworkspace.go

```diff
@@ -177,6 +177,11 @@ func (h *Handler) cleanupWorkspaceForDemotionToPowerUser(ctx context.Context, cl
 			}
 		}
 
+		workspace.Status.DefaultAccessControlRuleGenerated = false
```

Comment:
nit: only update if we know the value is changing.

Reply:
fixed